tests/lib.rs

```diff
@@ -156,15 +160,18 @@ fn foo() {
     let expected1 = "Hello, world!".to_owned();
     let expected2 = "Bye!".to_owned();
     let path = store.root().join("foo.txt");
+    let ctx = &mut ();
```

Comment:
I guess `warmy` would need a constant `const noctx: &'static mut () = &mut ()` or something like that. :)

Reply:
Yeah it is a common enough case that some typedef's wraping it would be nice.